src/price/mean.ts

```diff
@@ -0,0 +1,11 @@
+import assert from 'assert';
+
+export const mean = (values: number[]): number => {
+  assert(values.length > 0);
```

Comment:
I would better return `undefined` and do not stop code from executing. we can filter out undefined and still be able to submit what we have.

Reply:
we can still handle exceptions if we want to process it no matter what. I would leave this for now in case if we feed the function with empty input, which will never happen.